SalesforceCommerceCloud/pwa-kit
# Set cache control for home page

# Description
Pwa kit home page is reported to have excessive load times. This is because it is not cached.

The solution is to set the cache control on the home component that would allow it to result in CDN cache hits

GUS: https://gus.lightning.force.com/a07EE000001qo3RYAQ




# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- app/component/_app/index.jsx

# How to Test-Drive This PR

- Visit the deploy version here: https://scaffold-pwa-test.mobify-storefront.com/
- Confirming that there is a cache-control in header request
- Reload the home page a few times and confirm that the latter loading times are faster 

- If you want to do the benchmark test, you can run `ab -n 1000 -c 100 https://scaffold-pwa-test.mobify-storefront.com/` and compare the result with the one reported from the ticket

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
